python_package/surrealdb/ws.py

```diff
@@ -598,6 +600,7 @@ async def patch(
                 { 'op': "remove", "path": "/temp" },
             ])
         """
+        data = data or []
```

Comment:
This is for backwards compatibility, making the chance non-breaking.
The typing was Optional, but the server requires an array, `None` is invalid.